### ISSUE TITLE
v0.22 difference

### ISSUE THREAD

**Issue by jonny7737** [Chainfire/UselethMiner#8]
I noticed the following pattern in the console output

```
[2022-01-18 15:00:33][account.M1Mini-C@us1.ethermine.org:4444(15)][SUBMIT] epoch:1d3 header:4b1298080c1a7f67b291ef4236177af7d53335a137fa3b88266b7f46d7a783f0 hash:000000008345d054cd896816e350b34310f9434d7368e75698dd5475c2c47dd4 nonce: bd1800000066590 difficulty:1.95011/1.00000
[2022-01-18 15:00:33][account.M1Mini-C@us1.ethermine.org:4444(15)][ERROR] Provided PoW solution is invalid!
[2022-01-18 15:00:33][CFETHASH][2] Releasing threads
[2022-01-18 15:00:33][CFETHASH][2] Released threads
[2022-01-18 15:00:33][CFETHASH][3] Releasing threads
[2022-01-18 15:00:33][CFETHASH][3] Released threads
[2022-01-18 15:00:33][CFETHASH][DAG][0] Releasing DAG
[2022-01-18 15:00:33][CFETHASH][DAG][0] Released DAG
[2022-01-18 15:00:33][account.M1Mini-C@us1.ethermine.org:4444(15)] Connection lost
[2022-01-18 15:00:41][account.M1Mini-C@us1.ethermine.org:4444(20)] Connected
[2022-01-18 15:00:41][account.M1Mini-C@us1.ethermine.org:4444(20)][SUBSCRIBE] session:a432 extranonce:a432
[2022-01-18 15:00:41][account.M1Mini-C@us1.ethermine.org:4444(20)][UPDATE] difficulty:1.00000000
[2022-01-18 15:00:41][account.M1Mini-C@us1.ethermine.org:4444(20)][AUTHORIZE] username:account.M1Mini-C
[2022-01-18 15:00:43][FAILOVER] Switching to account.M1Mini-C@us1.ethermine.org:4444, priority 0
[2022-01-18 15:00:43][CFETHASH][DAG][0] Loading DAG
[2022-01-18 15:00:54][CFETHASH][DAG][0] Loaded DAG (hugepages:no)
[2022-01-18 15:00:54][CFETHASH][2] Initializing threads
[2022-01-18 15:00:54][CFETHASH][2] Initialized threads
[2022-01-18 15:00:54][CFETHASH][2] Setting threads to 7
[2022-01-18 15:00:54][CFETHASH][3] Initializing threads
[2022-01-18 15:00:54][CFETHASH][3] Initialized threads
```

This is the complete log sequence [no log entries deleted]

Notice the [ERROR] reply for invalid PoW solution.  This has happened several times since upgrading to v0.22 earlier today and did not happen prior to v0.22

Is this expected behaviour?

**Comment by Chainfire** [comment]
Definitely not... which flavors are being used here?

**Comment by Chainfire** [comment]
Try v0.23, should solve the problem. Seems something went wrong in the build process on v0.22.

**Comment by jonny7737** [comment]
Yes.  This issue is resolved by v0.23.

Thank you.
